src/ssl.c

```diff
@@ -32203,6 +32532,56 @@ static int EncryptDerKey(byte *der, int *derSz, const EVP_CIPHER* cipher,
 }
 #endif /* WOLFSSL_KEY_GEN || WOLFSSL_PEM_TO_DER */
 
+#ifndef NO_RSA
+static int wolfSSL_RSA_To_Der(WOLFSSL_RSA* rsa, byte** outBuf, int publicKey)
```

Comment:
Use wolfSSL_i2d_RSAPrivateKey() and wolfSSL_i2d_RsaPublicKey().
I think those functions are the real ones and we don't want to have public key code and private key code being include regardless of usage.

Reply:
Can you take a look at what I did with these functions? Done in commit https://github.com/wolfSSL/wolfssl/pull/2733/commits/f07c1f947fdc4480f6a5ae0ed2e5927773486f36

Reply:
It's not going to matter anyway as the OpenSSL APIs pull everything in.